.travis.yml

```diff
@@ -23,9 +23,8 @@ env:
 jobs:
     include:
         - stage: E2E Test
-          script:
-            ./scripts/test-e2e-ama.sh -host "localhost" -port 4100 -u "$E2E_USERNAME" -p "$E2E_PASSWORD" --unauthorized-user "$E2E_UNAUTHORIZED_USER" --unauthorized-user-password "$E2E_UNAUTHORIZED_USER_PASSWORD" -save -sh-url "$SCREENSHOT_URL" -sh-user "$SCREENSHOT_USERNAME" -sh-pass "$SCREENSHOT_PASSWORD" -dev
-
+          script: npm run e2e
```

Comment:
Package.json was modified, but the webdriver update doesn't happen this way. Please see the angular.json and this page:
https://angular.io/cli/e2e
`webdriverUpdate` has to be true in angular.json (or --webdriverUpdate=true from command line) for webdriver updating happen automatically.

Reply:
Default option for --webdriverUpdate is True, what I understood is that the update happen automatically, we don't need to specify this option.
In Travis, the build execution shows the chrome-driver update in the logs.

Reply:
Sorry, my bad, you are right!